_includes/head_custom.html

```diff
@@ -1 +1,2 @@
 <link rel="shortcut icon" href="https://raw.githubusercontent.com/f3d-app/f3d/master/resources/logo.ico" type="image/x-icon">
+{% include analytics.html %}
```

Comment:
why not putting the code directly instead of relying on an additional file?

Reply:
thats how the guide I've seen are doing it, but I can definitely change that if you want